cmd/manager/main.go

```diff
@@ -115,8 +125,19 @@ func main() {
 
 	log.Info("registering webhooks to the webhook server")
 	hookServer.Register("/mutate-pods", &webhook.Admission{Handler: &pod.Mutator{}})
-	hookServer.Register("/validate-inferenceservices", &webhook.Admission{Handler: &inferenceservice.Validator{}})
-	hookServer.Register("/mutate-inferenceservices", &webhook.Admission{Handler: &inferenceservice.Defaulter{}})
+
+	if err = ctrl.NewWebhookManagedBy(mgr).
```

Comment:
If you wanted to be fancy, you could wrap all this initialization logic in a for loop:

for resource, reconciler := range map[interface{}]Reconciler
} { 
  // register controller
  // register webhook
}